test/e2e/specs/specs-playwright/wp-theme__details-preview-activate.js

```diff
@@ -30,22 +32,32 @@ describe( DataHelper.createSuiteTitle( 'Theme: Activate' ), () => {
 		await sidebarComponent.gotoMenu( { item: 'Appearance', subitem: 'Themes' } );
 	} );
 
-	it( 'Search for free theme', async function () {
+	it( `Search for free theme with keyword ${ themeName }`, async function () {
 		themesPage = await ThemesPage.Expect( page );
 		await themesPage.filterThemes( 'Free' );
 		await themesPage.search( themeName );
 	} );
 
-	it( `Select a theme starting with ${ themeName }`, async function () {
-		await themesPage.select( themeName );
+	it( `Select and view details of a theme starting with ${ themeName }`, async function () {
+		const selectedTheme = await themesPage.select( themeName );
+		await themesPage.clickPopoverItem( selectedTheme, 'Info' );
```

Comment:
Just as a double check: do you think we're losing anything in terms of useful coverage by using the popover here instead of clicking directly on the theme card itself? I imagine that most users use the latter more than former - do we want to reflect that in this test?

Reply:
I debated back and forth on this a bit before pushing this revision up. 

Yes, by only using the meatball menu this would indeed reduce coverage of what you and I suspect would be the more typical path for users to traverse. 

Let's restore the original method (hover then click) and use it in the `wp-theme__details-preview-activate` spec so we'd have at least one coverage.